ui/packages/components/src/utils/historyParser/types.ts

```diff
@@ -37,7 +37,8 @@ export type HistoryType =
   | 'StepScheduled'
   | 'StepSleeping'
   | 'StepStarted'
-  | 'StepWaiting';
+  | 'StepWaiting'
+  | 'StepInvokingFunction';
```

Comment:
`StepInvoked`?  shorter == better all round.  At scale (and we do hundreds of millions a month) bandwidth costs go up for us and our users.

Reply:
I can change the enum: https://github.com/inngest/inngest/blob/571accdd55e046fa18cf311c1e2c2e77742970c1/pkg/enums/history_type.go#L7-L26
Honestly I figured they were serialized as integers. 😅